plugins/ruby-foreman-hdm/debian/foreman_hdm.rb

```diff
@@ -0,0 +1 @@
+gem 'foreman_hdm', '0.1.4'
```

Comment:
this file is unused (you have a `foreman_hdm.rb` outside the `debian` folder too)